homeassistant/components/kmtronic/__init__.py

```diff
@@ -26,7 +34,7 @@
 
 async def async_setup(hass: HomeAssistant, config: dict):
     """Set up the kmtronic component."""
-    hass.data[DOMAIN] = {}
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
Component setup is only done once before `async_entry_setup`.
```suggestion
    hass.data[DOMAIN] = {}
```

Reply:
Curious, I actually saw this pattern in other components and decided to change to make it consistent

Reply:
We use `hass.data.setdefault(DOMAIN, {})` if it's in `async_setup_entry` and the integration can have more than one entry.

Reply:
But you can have more than one entry in this integration, what am I missing ?

Reply:
We're in `async_setup` which is only run once.

Reply:
gotcha